### PR TITLE
FOGL-2107 Updated README

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

**Review by Singhal-Vaibhav — CHANGES_REQUESTED** [review]
The instructions are still not clear from a end user perspective. We are building the freeopcua library using `make`. That does not make the libraries available globally. Since our plugin is dependent on these libraries, we should provide an option in our plugin to search for opcua files in a particular location

**Comment by MarkRiddoch** [comment]
Vaibhav, the object is not to "make the libraries available globally", it is to build the plugin.

Please elaborate on what is not clear here.

File: README.rst [hunk 1]
```diff
@@ -6,17 +6,40 @@ A simple asynchronous OPC UA plugin that registers for change events on OPC UA o
...
-Thia plugin assumes the freeopcua is available at a fixed location in the file system. To build you
-must clone the freeopcua repository in your home directory.
+This plugin assumes the freeopcua is available at a specified location in the file system. To build you
+must clone the freeopcua repository to a directory of your choice.
 
 .. code-block:: console
 
   $ git clone https://github.com/FreeOpcUa/freeopcua.git
+  $ cd freeopcua
+  $ export OPCUADIR=`pwd`
```
Comment (Singhal-Vaibhav) [inline README.rst]: As per the script `set(OPCUADIR $ENV{FREEOPCUA})` it should be `export FREEOPCUA=...`

**Review by Singhal-Vaibhav — CHANGES_REQUESTED** [review]
1. Change required in environment variable name as mentioned in comment
2. Getting these warnings:
```
CMake Warning (dev) at CMakeLists.txt:64 (add_library):
  Policy CMP0003 should be set before this line.  Add code such as

    if(COMMAND cmake_policy)
      cmake_policy(SET CMP0003 NEW)
    endif(COMMAND cmake_policy)

  as early as possible but after the most recent call to
  cmake_minimum_required or cmake_policy(VERSION).  This warning appears
  because target "opcua" links to some libraries for which the linker must
  search:

    common-lib, -lpthread, -ldl

  and other libraries with known full path:

    /home/foglamp/Development/freeopcua/build/lib/libopcuaprotocol.so

  CMake is adding directories in the second list to the linker search path in
  case they are needed to find libraries from the first list (for backwards
  compatibility with CMake 2.4).  Set policy CMP0003 to OLD or NEW to enable
  or disable this behavior explicitly.  Run "cmake --help-policy CMP0003" for
  more information.
This warning is for project developers.  Use -Wno-dev to suppress it.
```

After discussion with @AmandeepArora , if we use `cmake_minimum_required(VERSION 2.6.0)` in CMakeLists.txt, these warnings will go away

**Review by Singhal-Vaibhav — APPROVED** [review]
LGTM!!

**Pull request merged**